tests/pc/test_po_update.py

```diff
@@ -374,3 +376,214 @@ def del_add_members():
+        tbinfo,
+        ptfadapter,
+        reload_testbed_on_failed, localhost):
+    """
+       Test port channel traffic with higher lag ids.
+       """
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+
+    # Check if the device is a modular chassis and the topology is T2
+    is_chassis = duthost.get_facts().get("modular_chassis")
+    if not (is_chassis and tbinfo['topo']['type'] == 't2'):
+        # Skip the test if the setup is not T2 Chassis
+        pytest.skip("Test is Applicable for T2 Chassis Setup")
+
+    dut_mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
+
+    # Send initial data to the device
+    send_data(dut_mg_facts, duthost, ptfadapter)
+    # Get the supervisor node (suphost) for the modular chassis setup
+    suphost = duthosts.supervisor_nodes[0]
+    # Get the established BGP neighbors from the DUT
+    up_bgp_neighbors = duthost.get_bgp_neighbors_per_asic("established")
+
+    # Log information about cold reboot on the supervisor node
+    logging.info("Cold reboot on supervisor node: %s", suphost.hostname)
+
+    # Reboot the supervisor node and wait for critical processes to restart
+    reboot(suphost, localhost, wait=240, safe_reboot=True)
+    logging.info("Wait until all critical processes are fully started")
+    wait_critical_processes(suphost)
+
+    # Ensure all critical services have started on the supervisor node
+    pytest_assert(wait_until(330, 20, 0, suphost.critical_services_fully_started),
+                  "All critical services should fully started! {}".format(suphost.hostname))
+
+    # For each linecard (frontend node), wait for startup and critical processes to start
+    for linecard in duthosts.frontend_nodes:
+        wait_for_startup(linecard, localhost, delay=10, timeout=300)
+        dut_uptime = linecard.get_up_time()
+        logging.info('DUT {} up since {}'.format(linecard.hostname, dut_uptime))
+
+        logging.info("Wait until all critical processes are fully started")
+        wait_critical_processes(linecard)
+
+        # Ensure all critical services have started on the linecard
+        pytest_assert(wait_until(330, 20, 0, linecard.critical_services_fully_started),
+                      "All critical services should fully started! {}".format(linecard.hostname))
+
+    # Perform a sanity check on the LAG set
+    lag_set_sanity(duthosts)
+
+    # Increment LAG IDs up to 500
+    increment_lag_id(duthost, 500)
+
+    # Perform a Config Reload to put the new lag ids on Portchannel
+    config_reload(duthost, safe_reload=True)
+
+    # Ensure BGP sessions are re-established after reload
+    pytest_assert(wait_until(300, 10, 0,
+                             duthost.check_bgp_session_state_all_asics, up_bgp_neighbors, "established"))
+
+    # Perform another LAG set sanity check
+    lag_set_sanity(duthosts)
+
+    # Send data after the configuration reload
+    send_data(dut_mg_facts, duthost, ptfadapter)
```

Comment:
The lag in the minigraph will always use the higher lag-ids?

Reply:
@arlakshm 
There is no relation to minigraph here.
We are trying to increase the lag ids allocation in free_list by creating and deleting temp PC.
Please let me know if there is any other thoughts regarding this